src/reagent_forms/core.cljs

```diff
@@ -493,22 +493,31 @@
        (map? (second node))
        (contains? (second node) :field)))
 
-(defn bind-fields
-  "creates data bindings between the form fields and the supplied atom
-   form - the form template with the fields
-   doc - the document that the fields will be bound to
-   events - any events that should be triggered when the document state changes"
```

Comment:
Whooops, I probably shouldn't have removed that docstring! I'll bring it back in a followup PR with updates to the docs later today

Reply:
👍

Reply:
sounds like a plan, and I can cut a release after :)